README.md

```diff
@@ -1,6 +1,6 @@
 # Commodore Component: nfs-subdir-external-provisioner
 
-This is a [Commodore][commodore] Component for nfs-subdir-external-provisioner.
+This is a [Commodore][commodore] Component for the [Kubernetes NFS Subdir External Provisioner](https://github.com/kubernetes-sigs/nfs-subdir-external-provisioner) provided by the Kubernetes Community.
```

Comment:
Probably makes sense to note in the README that "NFS Subdir External Provisioner" used to be the nfs-client-provisioner

Reply:
Added